packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -284,6 +287,9 @@ const NuovaDelega = () => {
                               'selectTuttiEntiOrSelezionati',
                               event.currentTarget.value
                             );
+                            if(event.currentTarget.value === 'entiSelezionati') {
```

Comment:
Il requisito è che venga chiamata l'api solo la prima volta che l'utente fa lo switch.
In più (da requisito) va aggiunto il loading nella select